### PR TITLE
Issue-275: Honor existingAgentk8sglueSecret in deployment template

### PR DESCRIPTION
**Pull request by savitha-qs**
**What this PR does / why we need it**:

Addresses [Issue 275](https://github.com/allegroai/clearml-helm-charts/issues/275)

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/allegroai/clearml-helm-charts/blob/main/CONTRIBUTING.md#pull-requests) guide (**required**)
- [x] Verify the work you plan to merge addresses an existing [issue](https://github.com/allegroai/clearml-helm-charts/issues) (If not, open a new one) (**required**)
- [x] Check your branch with `helm lint` (**required**)
- [x] Update `version` in `Chart.yaml` according [semver](https://semver.org/) rules (**required**)
- [x] Substitute `annotations` section in `Chart.yaml` annotating implementations (useful for Artifecthub changelog) (**required**)
- [x] Update chart README using [helm-docs](https://github.com/norwoodj/helm-docs) (**required**)

**Which issue(s) this PR fixes**:

Fixes #<275>

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by valeriano-manassero** [comment]
Checks looks good but there's still missing annotations as required in point 5 of 6

**Comment by savitha-qs** [comment]
@valeriano-manassero I've updated the annotation now. Let me know if it looks good. Thanks!

File: charts/clearml-agent/Chart.yaml [hunk 1]
```diff
@@ -22,3 +22,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
```
Comment (valeriano-manassero) [inline charts/clearml-agent/Chart.yaml:23]: pls remove this and following line since they are not part of this release

**Review by valeriano-manassero — DISMISSED** [review]

**Comment by savitha-qs** [comment]
@valeriano-manassero If this looks good to you, would you mind approving the PR? Or, feel free to merge it. Would appreciate it, thanks!

**Pull request merged**